portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesOOTB.testcase

```diff
@@ -87,6 +87,84 @@ definition {
 		}
 	}
 
+	@description = "LPS-134466 When removes the default link within Instance Settings/Asserts/Categorization, the message 'Learn how' won't show in web content editor"
+	@priority = "3"
+	test CheckLearnHowLinkNotDisplayedWCEditor {
+		task ("Remove default link to Documentation") {
+			Category.removeLinkToDocumentationURL();
+		}
+
+		task ("Assert if Learn How is not displayed") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+			WebContentNavigator.gotoAddCP();
+
+			AssertElementNotPresent(locator1 = "VOCABULARY_LINK_DOCUMENTATION");
+		}
+	}
+
+	@description = "LPS-134466 When removes the default link within Instance Settings/Asserts/Categorization, the message 'Learn how' won't show in Vocabularies Management"
+	@priority = "3"
+	test CheckLearnHowLinkNotDisplayedVocabularies {
+		task ("Remove default link to Documentation") {
+			Category.removeLinkToDocumentationURL();
+		}
+
+		task ("Assert if Learn How is not displayed") {
+			Category.openCategoriesAdmin(siteURLKey = "global");
+
+			AssertElementNotPresent(locator1 = "VOCABULARY_LINK_DOCUMENTATION");
+		}
+	}
+
+	@description = "LPS-134466 When removes the default link within Instance Settings/Asserts/Categorization, the message 'Learn how' won't show in Content Dashboard"
+	@priority = "3"
+	test CheckLearnHowLinkNotDisplayedContentDashboard {
+		task ("Remove default link to Documentation") {
+			Category.removeLinkToDocumentationURL();
+		}
+
+		task ("Assert if Learn How is not displayed") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Content",
+				panel = "Applications",
+				portlet = "Content Dashboard");
+
+			AssertElementNotPresent(locator1 = "VOCABULARY_LINK_DOCUMENTATION");
+		}
```

Comment:
Don't you need to put the path file before? like "Category#VOCABULARY_LINK_DOCUMENTATION"??